samples/NHSD.BuyingCatalogue.Identity.Api.SampleClient/Program.cs

```diff
@@ -19,7 +19,7 @@ private static async Task Main()
                 Console.WriteLine(discoveryDocument.Error);
```

Comment:
Are there no changes in this file?

Reply:
This could have been a merge conflict. Looks like tabbing.